tensorflow_addons/image/BUILD

```diff
@@ -164,3 +165,12 @@ py_test(
         ":image",
     ],
 )
+
+py_test(
+    name = "augment_ops_test",
+    size = "medium",
+    srcs = [
+        "augment_ops_test.py",
+    ],
+    main = "augment_ops_test.py",
+)
```

Comment:
```suggestion
    main = "augment_ops_test.py",
    deps = [
        ":image",
    ],
)
```

Reply:
The build fails due to a syntax error in `configure.py` and hence then Bazel test too.

Reply:
@gabrieldemarmiesse if you can help me with the failing Bazel Test. Seems like it's an import error. I've just used pytest to run tests locally. Working on setting up docker but it's taking too long.

Reply:
Sure, I added the modifications to do in my first message. You need to add the `deps`:

```
    main = "augment_ops_test.py",
    deps = [
        ":image",
    ],
)
```

I know bazel can be quite fustrating. We need to keep it because googlers make heavy use of it.